rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -328,13 +329,40 @@ def test_price_underlying_tokens(inquirer, globaldb):
     assert price == FVal(67)
 
 
+@pytest.mark.vcr()
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
-@pytest.mark.parametrize('should_mock_current_price_queries', [True])
-def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager):
-    address = '0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager, globaldb):
+    """Tests that the Uniswap v2 lp token price is correctly found. The price calculation
+    requires the underlying tokens of the pool. Pool and lp token are the same contract"""
+    address = '0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'  # LINK ETH POOL
+    tuples = [
+        ('eip155:1/erc20:0x514910771AF9Ca656af840dff83E8264EcF986CA', 0.5, f'eip155:1/erc20:{address}'),  # noqa: E501
+        ('eip155:1/erc20:0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', 0.5, f'eip155:1/erc20:{address}'),  # noqa: E501
+    ]
```

Comment:
probably this will need to change

Reply:
as I mentioned 0.5 is only if the pool is balanced but doesn't need to be the case in general